custom_components/spotcast/helpers.py

```diff
@@ -28,15 +28,24 @@ def get_spotify_devices(hass, spotify_user_id):
                 isinstance(entity, SpotifyMediaPlayer)
                 and entity.unique_id == spotify_user_id
             ):
-                _LOGGER.debug(
-                    f"get_spotify_devices: {entity.entity_id}: {entity.name}: %s",
-                    entity._devices,
-                )
+                if hasattr(entity, "_devices"):
```

Comment:
modify to a try/except. The error type is `AttributeError`. I was thinking of something like that:

```python
try:
    devices_data = entity._devices
except(AttributeError):
    devices_data = entity.data.devices.data

_LOGGER.debug(f"get_spotify_devices: {entity.entity_id}: {entity.name}: %s", devices_data)
```